packages/snippet-bot/README.md

```diff
@@ -44,6 +44,16 @@ test.py:10, tag lol doesn't have a matching start tag
 test.py:8, tag world doesn't have a matching end tag
 ```
 
+## Comment on PRs
+
+The bot adds a comment summarizing changes w.r.t region tags in the
+current PR.
+
+## Refreshing on PRs
+
+You can add `snippet-bot:refresh` label on PRs for refreshing the
```

Comment:
WDYT about `snippet-bot:force-run`? It matches the style of `kokoro:force-run` and `release-please:force-run`

Reply:
Sounds good

Reply:
Note: that we should document these labels in the top level README perhaps, like @codyoss suggested.

Reply:
What's the original suggestion? Maybe I can file a new issue for it.